test/functional/legacy/059_utf8_spell_checking_spec.lua

```diff
@@ -0,0 +1,953 @@
+-- Tests for spell checking with 'encoding' set to "utf-8".
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+local write_file = helpers.write_file
+
+describe("spell checking with 'encoding' set to utf-8", function()
+  setup(function()
+    clear()
+    execute("syntax off")
+    -- This file should be encoded in ISO8859-1.
```

Comment:
Both base64 and `\x??` (in lua) are ways to specify _binary_ data.  Base64 might be more effective (fewer text bytes per binary bytes), but in both cases the encoding would be equally "manual".
an alternative, if that happens to work, might be change to `SET UTF-8\n` and just store the text as utf-8.